Sprint-3/implement/rotate-char.test.js

```diff
@@ -0,0 +1,49 @@
+// rotateCharacter.test.js
+const rotateCharacter = require('./rotate-char.js'); // Import the function
+
+describe('rotateCharacter', () => {
+    // Test for lowercase letters
+    test('should rotate lowercase letter "a" by 3 positions', () => {
```

Comment:
There are other interesting cases that should be tested. In particular, a wider shift value range.